excellence-fee/excellence-fee.php

```diff
@@ -81,6 +81,7 @@ function excellence_submenu() {
 }
 function excellence_submenu_callback() {
     echo '<h3>Excellence Fee Addition</h3>';
+     include "admin/admin_input.php";
```

Comment:
@Shraddha1596 
why include in function, is ithis something new practice ?  correct it